README.md

```diff
@@ -9,18 +9,47 @@ Use links instead.
 -->
 
 # is-charms-template
+<!-- Use this space for badges -->
 
-Charmhub package name: operator-template
-More information: https://charmhub.io/is-charms-template
+Describe your charm in 1-2 sentences. Include the software that the charm deploys (if applicable), and the substrate (VM/K8s)
 
-Describe your charm in one or two sentences.
+If this is a bundle, list its components.
```

Comment:
Bundles have been officially deprecated, so it think it'd be better to remove this

Reply:
Sounds good, will remove